controllers/common/types.go

```diff
@@ -36,6 +38,20 @@ const (
 	AnnotationCleanFinalizerForced = `forced`
 )
 
+var log = ctrl.Log.WithName("controller")
+
+//ControllerCfg is a global variable to keep the configuration for Chaos Controller
+var ControllerCfg *config.ChaosControllerConfig
+
+func init() {
+	conf, err := config.EnvironChaosController()
```

Comment:
At present, we also have some configuration defined by the flag. Maybe we need to manage all configuration in this way and we also can unify this in a single pr.

Reply:
> At present, we also have some configuration defined by the flag. Maybe we need to manage all configuration in this way and we also can unify this in a single pr.

Got it. I will do it in a single PR.